node_zeromq/zmq.d.ts

```diff
@@ -95,6 +95,23 @@ declare module 'zmq' {
         bindSync(addr: string): Socket;
 
         /**
+         * Async unbind.
+         *
+         * Emits the "unbind" event.
+         *
+         * @param addr Socket address
+         * @param cb Unind callback
+         */
+        unbind(addr: string, callback: (error: string) => void ): Socket;
```

Comment:
I think callback is optional. https://github.com/JustinTulloss/zeromq.node/blob/master/lib/index.js#L318